src/rlookup.c

```diff
@@ -89,24 +230,20 @@ RLookupKey *RLookup_GetKeyEx(RLookup *lookup, const char *name, size_t n, int fl
   }
 
   if (!ret) {
-    ret = genKeyFromSpec(lookup, name, flags);
+    ret = genKeyFromSpec(lookup, name, n, flags);
```

Comment:
I could swear I made this change already in another PR... strange

Reply:
Oh it #3407, nevermind...

Reply:
It looked familiar to me as well 🤣